src/services/RetroGenerationService.ts

```diff
@@ -376,7 +381,8 @@ class RetroGenerationService {
    * @param testType
    */
   private isPassingLECTestType(testType: any): boolean {
-    const lecTestTypeIds = ["39", "201", "44", "45"];
+
+    const lecTestTypeIds = LEC_TEST.IDS;
```

Comment:
Is the local const declaration required here? 

The helper method in the commons was created so you can simply...

```
private isPassingLECTestType(testType: any): boolean {
  return TestTypeHelper.validateTestTypeIdInList(LEC_TEST, testType.testTypeId) && testType.testResult === TEST_RESULT_STATES.PASS;
}
```